pkg/util/crypt/crypt_dev_test.go

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
```

Comment:
Better just to use `2019`, since this file is created in 2019

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```